MsWheaPkg/Test/UnitTests/MsWheaEarlyStorageUnitTestApp/MsWheaEarlyUnitTestApp.c

```diff
@@ -601,15 +601,41 @@ MsWheaEarlyUnitTestAppEntryPoint (
     NULL
     );
 
-  AddTestCase (
-    Misc,
-    "Free ES slot find",
-    "MsWhea.Miscellaneous.MsWheaESFindSlotTest",
-    MsWheaESFindSlotTest,
-    MsWheaESVerify,
-    MsWheaESCleanUp,
-    NULL
-    );
+  // Only run the following tests if the early storage capacity is large enough
+  if (MsWheaEarlyStorageGetMaxSize () >= (sizeof (MS_WHEA_EARLY_STORAGE_HEADER) + (sizeof (MS_WHEA_EARLY_STORAGE_ENTRY_V0)))) {

```

Comment:
nit: Is there a reason you surrounded one `sizeof()` invocation with `()` but not the other? It looks a little strange unbalanced.

Reply:
Just a copy-paste artifact